rules/config

```diff
@@ -56,6 +56,10 @@ DEFAULT_PASSWORD = YourPaSsWoRd
 # Uncomment next line to enable:
 # INSTALL_DEBUG_TOOLS = y
 
+# SONIC_USE_PDDF_FRAMEWORK - Use PDDF generic drivers and plugins
+# Uncomment next line to enable:
+SONIC_USE_PDDF_FRAMEWORK = y
+
```

Comment:
is this intended to be enabled by default in the build ?
If this is disabled will the package be built into installer and still be installed into target ?

Reply:
if this is disabled, package will not be build.